lighthouserc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
might not need this file for this PR since we split off the a11y/lighthouse stuff